tutorials/send-email-with-sparkpost-and-nodejs-on-google-app-engine.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Send Email with SparkPost and Node.js on Google App Engine Flexible Environment
+description: Learn how to send email via SparkPost from a Node.js app to Google App Engine flexible environment.
+author: ewandennis
+tags: App Engine, SparkPost, Express.js, Node.js
+date_published: 2017-03-22
+---
+## SparkPost
+
+> [SparkPost][sparkpost]—Email delivery services built for developers, by developers.
+>
+> – sparkpost.com
+
+You can check out [Node.js and Google Cloud Platform][nodejs-gcp] to get an
+overview of Node.js itself and learn ways to run Node.js apps on Google Cloud
+Platform.
+
+## Prerequisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
```

Comment:
Consider mentioning that they should make note of the project ID created in this step, which they will need to swap in at the end where it says http://`YOUR_PROJECT_ID`.appspot.com